hibernate-reactive-core/src/main/java/org/hibernate/reactive/adaptor/impl/ResultSetAdaptor.java

```diff
@@ -214,7 +214,20 @@ public int getInt(String columnLabel) {
 
 	@Override
 	public long getLong(String columnLabel) {
-		Long aLong = row.getLong( columnLabel );
+		// PostgreSQL stores sequence metadata in information_schema as Strings.
+		// First try to get the value as a Long; if that fails, try to get
+		// as a String and try to parse it as a Long.
+		Long aLong;
+		try {
+			aLong = row.getLong(columnLabel);
+		}
+		catch (ClassCastException ex) {
+			// Check if the value is a String that can be converted to a Long
+			final String aString = row.getString(columnLabel);
+			// aString won't be null; check just because...
+			aLong = aString != null ? Long.parseLong(aString) : null;
+		}
```

Comment:
@gbadner won't this swallow the original CCE and cause it to be rethrown as something potentially misleading if it's *not* a string?

Reply:
Yes, you're right! I'll fix it to throw the original CCE in that case.

Reply:
Perfect, thanks 🙏

Reply:
@gavinking, I pushed a [commit](https://github.com/hibernate/hibernate-reactive/pull/903/commits/a4ec4b3e14b75639acbef9c8ff20a384534f20e6) to fix this. Does it look OK to you?

Reply:
Yup that's fine.